data/data/gcp/network/firewall.tf

```diff
@@ -22,7 +22,10 @@ resource "google_compute_firewall" "mcs" {
     ports    = ["22623"]
   }
 
-  source_ranges = [var.network_cidr]
```

Comment:
maybe fold this onto https://github.com/openshift/installer/blob/99aee21beb6f517ae9be83d90c91b7dc4565a360/data/data/gcp/network/firewall.tf#L60 ?

Reply:
Def fold this into control_plane.... one less firewall.

Reply:
thanks. done!